solana/program_v2/packages/tests/src/gatekeeper-suite/create-gatekeeper.test.ts

```diff
@@ -123,6 +107,48 @@ describe('Gateway v2 Client', () => {
           .rpc()
       ).to.eventually.be.rejectedWith(/InsufficientAuthKeys/);
     }).timeout(10000);
+
+    it('Create a gatekeeper with applied token fees', async function () {
```

Comment:
Is there a reason to use an anonymous function instead of an arrow function?

Reply:
I think only when we need to start using the this keyword since context is bound. 
But I don't think there is a reason to use here other than preference.